git-hooks/pre-push

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env php
+
+<?php
+echo "Running npm run lint...";
+exec('npm run lint', $output, $returnCode);
+if (count($output) !== 8) { // Hard coded array length to 8 since return code is empty
```

Comment:
I used your style. Now the linter won't allow to push even if there is a warning. The current codebase doesn't have any warnings so it shouldn't be a problem